Cargo.toml

```
@@ -2,6 +2,7 @@
 authors = ["Matthew Treinish <mtreinish@kortar.org>"]
 categories = ["development-tools::testing"]
 description = "A subunit v2 protocol implementation in Rust"
+edition = "2021"
```

We probably should set an explicit MSRV here if we're going to start using a version.

```suggestion
edition = "2021"
rust-version = "1.56"
```

---

An MSRV can be very nice for people wanting to know whether they will be able to compile successfully or not; could we take that conversation separately though? 1.56 is verging on ancient, and since rust-version implies a working lower-bound, we might want to get things looking nice and the decide what the lower bound we can easily support is.

---

Lets leave it as is then, we can pick a more recent msrv later if the need arises.